packages/calcite-components/src/components/shell-panel/shell-panel.e2e.ts

```diff
@@ -118,7 +118,7 @@ describe("calcite-shell-panel", () => {
       "calcite-shell-panel",
       (panel: HTMLCalciteShellPanelElement, containerClass: string, contentClass: string) => {
         const container = panel.shadowRoot.querySelector(containerClass);
-        return container.firstElementChild.className == contentClass;
+        return container.firstElementChild.classList.contains(contentClass);
```

Comment:
In Lumina, when `class={}` prop is an object, the attribute is set to not just `"value"` but to `" value "` (notice the spaces.
Adjusting this to use classList.contains to have the code work both pre and post migration